modules/oc-mirror-installing-plugin.adoc

```diff
@@ -56,9 +66,31 @@ $ sudo mv oc-mirror /usr/local/bin/.
 
 .Verification
 
+ifndef::oc-mirror[]
```

Comment:
These conditionals still aren't right. You need to look at the previews and compare the v2 assembly vs the v1 assembly:

* https://77247--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/about-installing-oc-mirror-v2.html#installation-oc-mirror-installing-plugin_installing-mirroring-disconnected
* https://77247--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected#installation-oc-mirror-installing-plugin_installing-mirroring-disconnected

The both are showing the `$ oc mirror --v2 --help` one.

You'll need to troubleshoot this a bit. But keep in mind that you use `ifndef` to mean to do the following only if the attribute is _not_ defined. If you want to do something if the attribute _is_ defined, then you use `ifdef`.

But either way, you need to test and compare the previews to make sure things are appearing as expected in v1 vs v2.